src/main/java/com/pmm/budgetcalculator/expensing/entity/Expense.java

```diff
@@ -23,19 +25,20 @@
 
 public class Expense extends BaseEntity {
 
-    @NotBlank(message = "this field is mandatory")
+//    @NotBlank(message = "this field is mandatory")
     private String expenseName;
     private String expenseDescription;
     @DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
     private Date expenseTime;
     @NotNull
     private BigDecimal expenseAmount;
-    @Size(min = 1, max = 30, message = "About Me must be between 1 and 30 characters")
+    @Size(min = 1, max = 30, message = "Must be between 1 and 30 characters")
     private String expensePlace;
-    @OneToMany(mappedBy = "expense", cascade = CascadeType.PERSIST)
+    @OneToMany(mappedBy = "expense", cascade = CascadeType.PERSIST, orphanRemoval = true)
+    @JsonManagedReference
```

Comment:
Remove this adnotation and include it with DTO with proper mapping in mapper via mapstruct.